### PR TITLE
Api 5.0 WP 9.5 (Google Place Venues)

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
TBH I didn't try to send `InputVenueMessageContent` and `ILQRVenue` with both foursquare and google place details, but as that doesn't seem to work to `send_venue`, I assume it doesn't for those either …

### CODE REVIEW

File: telegram/inline/inlinequeryresultvenue.py [hunk 1]
```diff
@@ -41,8 +45,8 @@ class InlineQueryResultVenue(InlineQueryResult):
         address (:obj:`str`): Address of the venue.
         foursquare_id (:obj:`str`): Optional. Foursquare identifier of the venue if known.
         foursquare_type (:obj:`str`): Optional. Foursquare type of the venue, if known.
-            (For example, "arts_entertainment/default", "arts_entertainment/aquarium" or
-            "food/icecream".)
```
Comment (Poolitzer) [inline telegram/inline/inlinequeryresultvenue.py:45]: hmm I like this actually. Why did you remove it, maintaining?

**Review by Poolitzer — COMMENTED** [review]
just one question, changes look great

File: telegram/inline/inlinequeryresultvenue.py (hunk 1 above)
Comment (Bibo-Joshi) [reply telegram/inline/inlinequeryresultvenue.py:45]: Because as a rule of thumb, attribute docstrings should be shorter than the corresponding argument docstring. at least that's what I did up to now …
